internal/core/data/v2/bootstrap/handlers/database/database.go

```diff
@@ -0,0 +1,131 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package db
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/edgexfoundry/edgex-go/internal/core/data/v2/bootstrap/container"
+	dbInterfaces "github.com/edgexfoundry/edgex-go/internal/core/data/v2/infrastructure/interfaces"
+	"github.com/edgexfoundry/edgex-go/internal/core/data/v2/infrastructure/redis"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/bootstrap/interfaces"
+	"github.com/edgexfoundry/edgex-go/internal/pkg/db"
+
+	bootstrapContainer "github.com/edgexfoundry/go-mod-bootstrap/bootstrap/container"
+	"github.com/edgexfoundry/go-mod-bootstrap/bootstrap/startup"
+	bootstrapConfig "github.com/edgexfoundry/go-mod-bootstrap/config"
+	"github.com/edgexfoundry/go-mod-bootstrap/di"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+)
+
+// httpServer defines the contract used to determine whether or not the http httpServer is running.
+type httpServer interface {
+	IsRunning() bool
+}
+
+// Database contains references to dependencies required by the database bootstrap implementation.
+type Database struct {
+	httpServer httpServer
+	database   interfaces.Database
+	isCoreData bool
```

Comment:
Why do we need this?

Reply:
In v1 implementation, the `isCoreData` boolean is used to create a new CoreData db client in https://github.com/edgexfoundry/edgex-go/blob/c5134682d316d7e5a1de6694786c0c16bb482985/internal/pkg/db/redis/client.go#L45-L57
We might still need this for V2 API implementation.

Reply:
Please remove and add back when actually needed in V2.

Reply:
Removed the `isCoreData` bool in `Database` struct of `internal/core/data/v2/bootstrap/handlers/database/database.go`, and replaced `CoreDataClient` struct with a general `Client` in `internal/core/data/v2/infrastucture/redis/client.go`.